### DISCUSSION TITLE
note/vuepress-theme-plume/theme-config/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/pengzhanbo.cn#38] [Blog-Comment]
# note/vuepress-theme-plume/theme-config/

依赖环境
Node.js v12+ (https://nodejs.org/en/); 
Yarn v1 classic (https://classic.yarnpkg.com/en/); ::: tip 提示
使用 pnpm (https://pnpm.io/zh/) 时，你需要在 .npmrc 文件中设置 shamefully-hoist=tru...

https://pengzhanbo.cn/note/vuepress-theme-plume/quick-start/

**Comment by WeiLin-Liao** [comment]
跑起来是默认主题，Welcome to your VuePress site

**Comment by WeiLin-Liao** [comment]
yarn 1.22.19

node v14.19.1

跑了好几次都是默认主题。







                       
原始邮件
                       
                     

发件人："giscus[bot]"< ***@***.*** &gt;;

发件时间：2023/8/9 17:40

收件人："pengzhanbo/pengzhanbo.cn"< ***@***.*** &gt;;

抄送人："WeiLin-Liao"< ***@***.*** &gt;;"Comment"< ***@***.*** &gt;;

主题：[pengzhanbo/pengzhanbo.cn] note/vuepress-theme-plume/theme-config/(Discussion #38)




 
note/vuepress-theme-plume/theme-config/
 
依赖环境
 Node.js v12+ (https://nodejs.org/en/);
 Yarn v1 classic (https://classic.yarnpkg.com/en/); ::: tip 提示
 使用 pnpm (https://pnpm.io/zh/) 时，你需要在 .npmrc 文件中设置 shamefully-hoist=tru...
 
https://pengzhanbo.cn/note/vuepress-theme-plume/quick-start/
